mbrl/diagnostics/training_browser.py

```diff
@@ -16,6 +16,7 @@
 from matplotlib.figure import Figure
```

Comment:
One more thing, if you haven't already, can you install https://pre-commit.com/ to run the linter, mypy, etc. automatically?